cmd/gcstools/installdrivers.go

```diff
@@ -13,76 +13,64 @@ import (
 
 	"github.com/Microsoft/hcsshim/internal/guest/storage/overlay"
 	"github.com/Microsoft/hcsshim/internal/log"
-	"github.com/google/uuid"
 	"github.com/pkg/errors"
 )
 
-const (
-	lcowGlobalDriversFormat = "/run/drivers/%s"
-
-	moduleExtension = ".ko"
-)
+const moduleExtension = ".ko"
 
 func install(ctx context.Context) error {
 	args := []string(os.Args[1:])
 
-	if len(args) == 0 {
-		return errors.New("no driver paths provided for install")
+	if len(args) < 2 {
+		return fmt.Errorf("expected two args, instead got %v", len(args))
```

Comment:
It would likely just mean that they either put nonesense after the driver or tried to specify multiple drivers at a time. Since in the shim we never issue this command with multiple driver paths, we don't really need to care here.